style.md

```diff
@@ -24,3 +74,4 @@ The exceptions are entries where the Go default for the type is a no-op in the s
 [no-pointer-for-slices]: https://github.com/opencontainers/specs/pull/316/files#r50782982
 [optional-pointer]: https://github.com/opencontainers/specs/pull/233#discussion_r47829711
 [pointer-when-updates-require-changes]: https://github.com/opencontainers/specs/pull/317/files#r50932706
+[markdown-headers]: https://help.github.com/articles/basic-writing-and-formatting-syntax/#headings
```

Comment:
I think [this][1] would be a more canonical link, but both are likely authoritative enough for this situation.

[1]: http://daringfireball.net/projects/markdown/syntax#header